### ISSUE TITLE
many option missing from fenix 

### ISSUE THREAD

**Issue by ale82to** [CookieJarApps/BiscuitBrowser#14]
I tried to switch from fenix but for now I encountered many issues.
I couldn't import violent monkey back up from fenix.
also many important options from fenix mostly fixed bottom toolbar and font size in accessibility are missing.
also the white url bar with dark theme is not very nice in my opinion.

**Comment by CookieJarApps** [comment]
The browser's still in fairly early stages of development, so that's why there's a lot missing, but I'm aiming by the first full release to have something functionally on-par with Fennec (Fenix is missing lots of useful features).

I'll definitely add your suggestions to the list for the next update, though.

**Comment by ale82to** [comment]
thank you for considering!

**Comment by CookieJarApps** [comment]
I've fixed all the issues mentioned in your issue in the code, and an update is coming tomorrow/the day after. Thanks for the suggestions!

**Comment by ale82to** [comment]
amazing thanks, looking forward to!
